src/main/java/com/uspray/uspray/infrastructure/SharedPrayRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.uspray.uspray.infrastructure;
+
+import com.uspray.uspray.DTO.pray.PrayDto;
+import com.uspray.uspray.domain.Member;
+import com.uspray.uspray.domain.SharedPray;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface SharedPrayRepository extends JpaRepository<SharedPray, Long>, SharedPrayRepositoryCustom {
+
+    // 수신자 기준 모두 찾기 (보관함 조회)
+    List<SharedPray> findAllByMemberOrderByCreatedAtDesc(Member member);
```

Comment:
```suggestion
    // 수신자 기준 모두 찾기 (보관함 조회)
    @EntityGraph(attributePaths = {"member"})
    List<SharedPray> findAllByMemberOrderByCreatedAtDesc(Member member);
```
연관된 테이블을 가져올 때 위처럼 EntityGraph를 명시해주면 테이블을 join해서 한방에 가져올 수 있습니다!